data/json/items/gun/308.json

```diff
@@ -58,8 +58,8 @@
     "id": "m134",
     "type": "GUN",
     "name": "M134D-H Minigun",
-    "description": "The M134 Minigun is a 7.62x51 mm NATO, six-barreled machine gun.  It features Gatling-style rotating barrels with an electric motor.  Its' recoil is too powerful for firing unmounted, but if you could attach it to a vehicle and find enough ammo for it, it would become a devastating weapon.",
-    "weight": 2450,
+    "description": "The M134-H Minigun is a lightweight, rotary machine gun, chambered for the 7.62x51mm cartridge.  Its barrels are rotated by an electric motor, powered by UPS.  Its weight, recoil and rotational torque is too high for firing unmounted, but if you could attach it to a vehicle and find enough ammo for it, it would become a devastating weapon.",
```

Comment:
At present nothing but the current UPS system is a mess so I'd prefer to avoid adding new instances using it until we can restore some sanity to it